Protos/swift-options.proto

```diff
@@ -3,7 +3,9 @@ syntax="proto2";
 
```

Comment:
Since the old version of descriptor.proto we had in this repo also had `swift_prefix`, we could just kill this file and the support for it since the .proto files will work with our old releases also.